src/gui/dialogs/unit_recall.cpp

```diff
@@ -146,6 +146,14 @@ static std::string get_title_suffix(int side_num)
 	return msg.str();
 }
 
+void unit_recall:unit_recall_default_compare(unit &first, unit &second) {
```

Comment:
Existing practice in Wesnoth codebase is to attach the `&` to the type name: `unit& first, unit& second`